packages/cli-lib/lib/config.js

```diff
@@ -301,7 +301,7 @@ const parseConfig = configSource => {
     return { parsed, error };
   }
   try {
-    parsed = yaml.safeLoad(configSource);
+    parsed = yaml.load(configSource);
```

Comment:
Why the move against `safe*`? I think it is best to be restrictive when it comes to serialization / deserialization of yaml since it can be used for more than just data if opened up.

Reply:
I was getting a deprecation warning, so switched it over.